packages/fhir-keycloak-user-management/src/components/CreateEditUser/tests/index.test.tsx

```diff
@@ -220,6 +227,20 @@ test('renders correctly for edit user', async () => {
   userEvent.click(markSupervisor);
 
   const submitButton = document.querySelector('button[type="submit"]');
+
+  // find antd Select with id 'practitioners' in the 'Form' component
+  const appIdSection = document.querySelector('[data-testid="fhirCoreAppId"]') as Element;
```

Comment:
Sorry, could you elaborate some more, what's the change you would like to happen here?

Reply:
Instead of using `document.querySelector` we should use a native test selector e.g `getByTestId('fhirCoreAppId')`. document queries the dom itself instead of whatever abstraction is created by jsdom

Reply:
there's more occurrence of this